charmcraft/logsetup.py

```diff
@@ -82,6 +82,11 @@ def ended_ok(self):
         """Cleanup after successful execution."""
         os.unlink(self._log_filepath)
 
+    def ended_interrupt(self):
```

Comment:
Please provide a test for this method. Thanks!

Reply:
done :-)